e2e/images/metrics-api/README.md

```diff
@@ -0,0 +1,22 @@
+# KEDA Sample for Metrics API
+
+
+A simple Docker container written in **go** that will expose metrics. This metrics will be used with for a Keda scaler [Metrics APi](https://keda.sh/docs/2.3/scalers/metrics-api/)
```

Comment:
Please, use KEDA instead of Keda and update the link to the latest version: https://keda.sh/docs/latest/scalers/metrics-api/